packages/office-ui-fabric-react/src/components/KeytipLayer/KeytipLayer.base.tsx

```diff
@@ -609,4 +600,30 @@ export class KeytipLayerBase extends BaseComponent<IKeytipLayerProps, IKeytipLay
     this.setState({ inKeytipMode: inKeytipMode });
     this._keytipManager.inKeytipMode = inKeytipMode;
   };
+
+  /**
+   * Emits a warning if duplicate keytips are found for the current keytip
+   */
+  private _detectDuplicateKeytips = (): void => {
+    const duplicateKeytips = this._getDuplicateIds(this._keytipTree.getChildren());
```

Comment:
What about duplications with persisted keytips? It looks like `getChildren` returns only non-persisted keytips

Reply:
Yeah I will take another look at this

Reply:
Looking at this more it would technically be redundant. A duplicate persisted keytip means there's a "regular" dupe keytip in the overflow menu. Opening that overflow menu will give you the warning. The consequence for executing the dupe is the same as if it was in the menu; it will only trigger the first one found. I think I will leave this as is